config/config.ini

```diff
@@ -49,6 +49,7 @@ GCWeatherURL = http://weather.gc.ca/city/pages/qc-147_metric_e.html
 GCWeatherAlertURL = https://weather.gc.ca/warnings/report_e.html?qc67
 WttrINTemplate = http://wttr.in/Montreal_2mpq_lang=en.png?_={}
 TepidURL = https://tepid.science.mcgill.ca:8443/tepid/screensaver/queues/status
+RecallChannel = food
```

Comment:
Can we rename this to "FoodRecallChannel" to be more descriptive?

Reply:
Sure thing